Docker/ods-api-bulk-load-console/alpine/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# SDK image used to install dotnet tool. Runtime image do not have 'dotnet tools' installed
+FROM mcr.microsoft.com/dotnet/sdk:8.0.201-alpine3.19@sha256:e646d8a0fa589bcd970e0ebde394780398e8ae08fffeb36781753c51fc9e87b0 AS builder
+
+ARG BULKLOAD_VERSION
+
+RUN dotnet tool install -g \
+    --add-source https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json \
+    EdFi.Suite3.BulkLoadClient.Console --version $BULKLOAD_VERSION && \
+    rm -f /root/.dotnet/tools/.store/edfi.suite3.bulkloadclient.console/$BULKLOAD_VERSION/edfi.suite3.bulkloadclient.console/$BULKLOAD_VERSION/tools/net8.0/any/log4net.config
+
+# Smaller image
+FROM mcr.microsoft.com/dotnet/runtime:8.0.2-alpine3.19@sha256:17c0535218406fa12788a26925c6d31e27cb52edf94c1aa7666f33ab804b1663
```

Comment:
| image | size |
| -- | -- |
|8.0.2-alpine3.19 |  82673615 |
| 8.0.2-jammy-chiseled | 85326482 | 

Interesting to see that Alpine is still slightly smaller than the new Ubuntu Chiseled images.

Reply:
Chiseled does not include apt nor bash, so we will need to find a better way to wait for pg_isready before starting the api container; for envsubst, we could use %variables% in the config template and use Environment.ExpandEnvironmentVariables() instead of it